Sources/MapboxMaps/Foundation/Extensions/UIWindow.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+@available(iOS 13.0, *)
+extension UIWindow {
+    private static let templateApplicationSceneSelector = Selector(("templateApplicationScene"))
```

Comment:
Let's declare constants in the narrowest scope possible. In this case, this can just be a local let constant in carPlayScene.